src/elements/content-sidebar/ActivitySidebar.js

```diff
@@ -53,6 +53,9 @@ type ExternalProps = {
     activeFeedEntryType?: FocusableFeedItemType,
     currentUser?: User,
     getUserProfileUrl?: GetProfileUrlCallback,
+    hasReplies?: boolean,
```

Comment:
Are we using this or is this for future changes?

Reply:
This is for future changes. More PRs will follow soon :)